### DISCUSSION TITLE
<div id="popup">

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3749] [Q&A]
For what is `<div id="popup">` used?
![grafik](https://user-images.githubusercontent.com/1645099/128253474-055936fb-e961-4d5e-95ea-16467637a10d.png)

**Comment by math-GH** [comment]
mhm, no answer :/

**Reply by Frenzie** [reply]
It's the `Article CSS selector on original website` popup to preview, but you can grep the source just as easily yourself. ;-)
